tests/expectations/snapshots/snap_test_expectation_atomic_renderers.py

```diff
@@ -963,16 +941,17 @@
                 "schema": {"type": "array"},
                 "value": ["my_first_column", "my_second_column"],
             },
-            "condition_parser": {"schema": {"type": "string"}, "value": None},
             "ignore_row_if": {"schema": {"type": "string"}, "value": "foo"},
             "mostly": {"schema": {"type": "number"}, "value": 0.8},
             "mostly_pct": {"schema": {"type": "string"}, "value": "80"},
-            "row_condition": {"schema": {"type": "string"}, "value": None},
-            "v__0": {"schema": {"type": "string"}, "value": "my_first_column"},
-            "v__1": {"schema": {"type": "string"}, "value": "my_second_column"},
+            "column_list_0": {"schema": {"type": "string"}, "value": "my_first_column"},
+            "column_list_1": {
+                "schema": {"type": "string"},
+                "value": "my_second_column",
+            },
```

Comment:
Another bug. `$column_list_0` would not have been substituted by the `v__0` param.